source/fe/mapping_p1.cc

```diff
@@ -0,0 +1,895 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2025 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE.md at
+// the top level directory of deal.II.
```

Comment:
Is this the correct copyright notice (I see this also in a few files below)? Or is it captured by our scripts anyway?